sda-commons-server-hibernate/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
   api 'org.flywaydb:flyway-core'
   api 'jakarta.annotation:jakarta.annotation-api'
   api 'org.flywaydb:flyway-core'
+  api 'org.flywaydb:flyway-database-postgresql'
```

Comment:
Would it be enough to just add it to the dependency management and check if it works without version in an example module?

Reply:
As Steve pointed out, the module is documented to [work specifically with PostgreSQL](https://github.com/SDA-SE/sda-dropwizard-commons/blob/ba635ebac3dd4d5885234a533d5c3ae07ff70095/docs/server-hibernate.md?plain=1#L6), so in my opinion it should provide everything needed to work out of the box.

Reply:
Hmmm, ok. Then let's keep it as it is here.